bot/util/utilities.py

```diff
@@ -1,29 +1,158 @@
+            .format(len(ctx.guild.roles))
+        str_features = generate_features_string(ctx.guild.features)
+
+        embed = discord.Embed(title=ctx.guild.name, timestamp=datetime.now(), color=0x1E90FF)
+        embed.set_thumbnail(url=ctx.guild.icon_url)
+        embed.set_footer(text="Erstellungsdatum")
+        embed.add_field(name="Besitzer :crown:", value=str_owner, inline=True)
+        embed.add_field(name="Server Boost <:server_boost:730390579699122256>", value=str_boosts, inline=True)
+        embed.add_field(name="Server Features :tools:", value=str_features, inline=True)
+        embed.add_field(name="Mitglieder :man_raising_hand:", value=str_members, inline=True)
+        embed.add_field(name="Kanäle :dividers:", value=str_channels, inline=True)
+        embed.add_field(name="Rollen :medal:", value=str_roles, inline=True)
+        await ctx.send(embed=embed)
+
+
+def determine_boost_level_cap(amount_boosts):
+    """Function for determining the current server level cap.
+
+    Args:
+        amount_boosts (int): The number of Boosts a server has received from its members.
+
+    Returns:
+        str: A short message indicating how many boosts are needed to level up.
+    """
+    level_cap = ""
+
+    if amount_boosts < 2:
+        level_cap = "/2"
+    elif 2 <= amount_boosts < 15:
+        level_cap = "/15"
+    elif 15 <= amount_boosts < 30:
+        level_cap = "/30"
+
+    return level_cap
+
+
+def get_channel_counters(guild):
+    """Function for counting the amount of different channels on a server.
+
+    Args:
+        guild (Guild): A Guild object which represents a Discord server.
+
+    Returns:
+        list: A list containing the total amount of channels, the amount of text channel and the amount of voice
+                channels on a server.
+    """
+    cntr_vc_channels = len(guild.voice_channels)
+    cntr_txt_channels = len(guild.text_channels)
+    cntr_channels = cntr_vc_channels + cntr_txt_channels
+
+    return [cntr_channels, cntr_txt_channels, cntr_vc_channels]
+
+
+def get_member_counters(guild):
+    """Function for counting the amount of members and bots on a server.
+
+    Args:
+        guild (Guild): A Guild object which represents a Discord server.
+
+    Returns:
+        list: A list containing the total amount of members, the amount of bots and the amount of human members.
+    """
+    cntr_bots = len(list(filter(lambda user: user.bot, guild.members)))
+
+    return [guild.member_count, cntr_bots, guild.member_count - cntr_bots]
+
+
+def generate_features_string(features):
+    """Function for creating a string which contains an enumeration of all available server features.
+
+    Args:
+        features (list): A list of available server features for a specific Discord server.
+
+    Returns:
+        str: A string containing an enumeration of all available server features.
+    """
+    ic_bullet_point = ":white_check_mark: "
+    str_features = ":no_entry_sign: Keine"
```

Comment:
Replace this and the if len()... in the next line with

if len(features) == 0:
   return ":no_entry_sign: Keine"